### PR TITLE
db: Update models to reflect latest schema updates

### PR DESCRIPTION
**Pull request by nilmerg**

### CODE REVIEW

File: library/Icingadb/Model/DependencyEdgeState.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        return [
+            'failed' => t('Dependency Edge State Failed')
+        ];
+    }
+
+    public function createBehaviors(Behaviors $behaviors): void
+    {
+        $behaviors->add(new BoolCast([
+            'failed'
+        ]));
```
Comment (raviks789) [inline library/Icingadb/Model/DependencyEdgeState.php:45]:
```suggestion
        $behaviors->add(new Binary([
            'id'
        ]));
        $behaviors->add(new BoolCast([
            'failed'
        ]));
```

**Review by raviks789 — COMMENTED** [review]
`is_reachable` column is missing in `RedundancyGroupState` model.

File: library/Icingadb/Model/DependencyEdgeState.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+use ipl\Orm\Behaviors;
+use ipl\Orm\Model;
+
+/**
+ * Dependency edge state model.
+ *
+ * @property string $id
+ * @property bool $failed
+ */
+class DependencyEdgeState extends Model
```
[the remark this answers is not in the document]
Comment (nilmerg) [reply library/Icingadb/Model/DependencyEdgeState.php:16]: done

**Comment by nilmerg** [comment]
> is_reachable column is missing in RedundancyGroupState model.

Will be added by #1073

**Review by raviks789 — APPROVED** [review]
LGTM

**Pull request merged**